components/youtube/new-videos-by-location.js

```diff
@@ -0,0 +1,75 @@
+const axios = require('axios')
+const youtube = require("https://github.com/PipedreamHQ/pipedream/components/youtube/youtube.app.js");
+
+module.exports = {
+  name: "Youtube - New Videos by Location",
```

Comment:
We're also adding this to all of our components, but can you add a `description` to these, as well? The format we like to use is something like:

"Emits an event for each new [X]", so in this case it can be something like "Emits an event for each new Youtube video tied to a location", then you can adjust for each source below.